packages/terra-clinical-item-view/src/terra-dev-site/doc/clinical-item-view/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,230 @@
+- Do not skip a heading level from the top down.
+
+More information on heading standards can be found [here](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/).
+
+Below you can find a couple examples of implementing a Terra Core Heading element above the Item View.
+
+</Notice>
+
+##### Code Examples
+
+<ItemViewWithHeader
+    title="ItemView - With Header"
+/>
+<ItemViewTwoColumnWithHeader
+    title="ItemView - Two Column Layout With Header"
+/>
+
+#### True Columns vs By Row
+
+The two column layout for the Clinical Item View has been updated to use true columns instead of rows formatted to look like columns. This affects not only the programmatic layout but also the screenreader's behavior in how these displays are read.
+The implementation of this is accomplished behind the `trueColumn` prop which defaults to true.
+
+The screenreader now reads through the first column, top to bottom, before moving on to the second column and reading its displays from top to bottom.
+This way of handling readout is better, as it is more natural for us to organize and read information in columns by reading them one at a time, top to bottom before reading through the next column.
+
+It is strongly recommended to utilize the true two column layout mentioned above, but if needed you can set the `trueColumn` prop to false which will revert to using the previous 'by row' logic.
+
+Some potential reasons you might do this:
+- When trueColumn is true, the spacing and truncation of elements are slightly different due to the fact that we are now splitting the displays into actual programmatic columns.
+We can't resize on a row by row basis between the two columns since they are separate, so truncation may start a little earlier than it did before.
+- When there is text wrapping/overflow for a display on one side - the 'by row' behavior adds blank newlines to the corresponding display in the opposite column.
+Alternatively, when we use the trueColumn layout and there is a display on one side with overflow, no blank lines are added below the display in the opposite column. The next display is rendered directly under the previous one with no blank lines.
+
+All of that being said, the screenreader readout for the 'by row' formatting announces the first display from the first column and then jumps over to the first display in the second column.
+It then jumps back to the first column and reads out the next display before moving back over to the second column and reading the next display. This pattern of reading displays by row continues until the last display is read.
+
+<Notice variant="important" ariaLevel="3">
+
+The primary reason this 'by row' formatting is not recommended is because visually when information is shown in a column it is natural to read all of the information in the first column and then the second.
+Non-sighted users should also have the opportunity to take in information this way.
+
+</Notice>
```

Comment:
+1 of the use of Notice here.
```suggestion
<Notice variant="important" ariaLevel="3">

The primary reason this 'by row' formatting is not recommended is because when information is displayed in columns, it is natural to read all of the information in the first column before moving on to the next one.
Assistive technology users should also have the opportunity to take in information this way.

</Notice>
```

Reply:
[e5cfead](https://github.com/cerner/terra-clinical/pull/916/commits/e5cfead1b5b012bd75f0ca2c002521340e341419)